src/pages/vrf/v2/subscription/examples/test-locally.mdx

```diff
@@ -0,0 +1,160 @@
+---
+layout: ../../../../../layouts/MainLayout.astro
+section: vrf
+title: "Local testing using a Mock contract"
+metadata:
+  description: "Example contract for generating random words using the VRF v2 subscription method on your local blockchain using a mock contract."
+---
+
+import VrfCommon from "@features/vrf/v2/common/VrfCommon.astro"
+import ContentCommon from "@features/common/ContentCommon.astro"
+import { CodeSample, ClickToZoom, Aside } from "@components"
+
+<VrfCommon callout="subscription" />
+
+This guide explains how to test Chainlink VRF v2 on a [Remix IDE](https://remix-ide.readthedocs.io/en/latest/run.html#environment) sandbox blockchain environment. **Note**: You can reuse the same logic on another development environment, such as hardhat or truffle. For example, read the hardhat starter kit [RandomNumberConsumer unit tests](https://github.com/smartcontractkit/hardhat-starter-kit/blob/main/test/unit/RandomNumberConsumer.spec.js).
```

Comment:
Capitalize these terms when referring to them in plain text (not code references): 

- "Hardhat"
- "Truffle"
- "Hardhat Starter Kit"

Apply throughout this document and the direct funding version.